internal/pkg/aws/sessions/errors.go

```diff
@@ -39,16 +39,11 @@ func (e *errCredRetrieval) Error() string {
 // RecommendActions returns recommended actions to be taken after the error.
 // Implements main.actionRecommender interface.
 func (e *errCredRetrieval) RecommendActions() string {
-	notice := "It looks like your credential settings are misconfigured or missing"
+	notice := "It looks like AWS credentials are misconfigured or missing"
 	if e.profile != "" {
-		notice = fmt.Sprintf("It looks like your profile [%s] is misconfigured or missing", e.profile)
+		notice = fmt.Sprintf("It looks like AWS profile [%s] is misconfigured or missing", e.profile)
 	}
 	return fmt.Sprintf(`%s:
-https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials
-- We recommend including your credentials in the shared credentials file.
-- Alternatively, you can also set credentials through 
-	* Environment Variables
-	* EC2 Instance Metadata (credentials only)
```

Comment:
Why do we remove these 🤔

Reply:
Documentation link has ll details and for someone who is using copilot for first time and doesnt have relevant experience of dealing with  aws  from terminal — those don’t mean match without context. In fact could give a wrong impression that one need to run copilot from within EC2 instance. 

If the goal for those messages to provide ideas on how someone who think they configured credentials and now need to debug missing credentials error - we should think more about this use case and how to help

Reply:
Gotcha makes sense to me.